README.md

```diff
@@ -26,3 +26,7 @@ which will generate one report for the whole run, or outside
 mpiexec -n 2 nsys profile --trace=nvtx,mpi --mpi-impl=openmpi --output=report.%q{OMPI_COMM_WORLD_RANK} julia --project mpi.jl
 ```
 which will generate a report for each MPI process, and can be opened as a "multi-report view".
+
+#### Example
+
+<img width="1904" alt="report1" src="https://user-images.githubusercontent.com/187980/182362221-aea7eb12-a736-406b-807f-1e1c46c406d0.png">
```

Comment:
Are these links permanent? Or do we need to upload it to the repo?

Reply:
I drag-n-dropped it into markdown so I would think they're permanent? But we can also put it in the repo of course.

Reply:
I guess we can try it: if it dies we can upload it elsewhere.